src/views/partials/scripts.hbs

```diff
@@ -1,2 +1,6 @@
+<script src="js/lib/q.min.js"></script>
+<script src="js/lib/chart.min.js"></script>
+<script src="js/recent.js"></script>
+<script src="js/engage.js"></script>
```

Comment:
If you go down that route, you could use Grunt + browserify as we do normally, or use Webpack. It's a pretty need build tool.